metametamodel/builtins.json

```diff
@@ -4,6 +4,10 @@
     {
       "key": "LionCore-M3",
       "version": "2023.1"
+    },
+    {
+      "key": "LionCore-builtins",
```

Comment:
Wasn't the dependency on this considered to be implicit?

Reply:
We discussed this several times and finally made this decision:
![image](https://github.com/LionWeb-io/specification/assets/818164/77ab238d-6df3-4a99-b920-e8c6709f6e05)
from the serialization document.

Reply:
Ah, right. Apologies for not looking this up in the spec. myself!